### PR TITLE
Doc(NOISSUE): Separate production and staging inventories

### PR DESCRIPTION
**Pull request by xFuture603**
Added documentation to separate your inventory by environment.

### CODE REVIEW

File: docs/separate_production_and_staging_inventory.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+```
+
+## Good
+
+Folder structure
+
+```bash
+/path/to/your/ansible
+├── ansible.cfg
+├── group_vars
```
Comment (rndmh3ro) [inline docs/separate_production_and_staging_inventory.md:45]: Should the group_vars and host_vars also live in the inventory-directories?

```
/path/to/your/ansible
├── ansible.cfg
├── group_vars
├── host_vars
├── inventory
│   ├── live
 |            |── group_vars
 |            |── host_vars
│   └── test
├── playbooks
├── roles
```

Comment (xFuture603) [reply docs/separate_production_and_staging_inventory.md:45]: Should they? I don't know if you should nest them in an inventory folder. @schurzi what do you think about this?

Comment (rndmh3ro) [reply docs/separate_production_and_staging_inventory.md:45]: It's definitely possible. This way, you separate the variables for the environments, too. Of course this also means duplication (e.g. in `all.yml`).

Comment (xFuture603) [reply docs/separate_production_and_staging_inventory.md]: Makes sense to me. I've added your suggestion to the documentation.

**Review by rndmh3ro — APPROVED** [review]

**Pull request merged**